script_editor/app/views/plays/show.html.erb

```diff
@@ -394,6 +396,7 @@
                     element.style.color = "#000000"
                     element.style.textDecoration = "none"
                     element.dataset.cut = "false"
+                    element.style.backgroundColor = '#FFFFFF'
```

Comment:
Why is the background color set as the same for cut and uncut elements?

Reply:
This background color is just to undo the highlighting background color, it's not actually a cut vs uncut thing. Putting it here prevents that weird thing where stuff would get cut right but stay highlighted